nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestEncodeContent.java

```diff
@@ -16,65 +16,50 @@
  */
 package org.apache.nifi.processors.standard;
 
+import static org.junit.Assert.assertEquals;
+
 import java.io.IOException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
+import org.apache.nifi.components.DescribedValue;
+import org.apache.nifi.processor.Relationship;
+import org.apache.nifi.processors.standard.encoding.EncodingMode;
+import org.apache.nifi.processors.standard.encoding.EncodingType;
+import org.apache.nifi.processors.standard.encoding.LineOutputMode;
 import org.apache.nifi.util.MockFlowFile;
+import org.apache.nifi.util.StringUtils;
 import org.apache.nifi.util.TestRunner;
 import org.apache.nifi.util.TestRunners;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public class TestEncodeContent {
 
+    private static final String LOREM_IPSUM = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.";
+
     private static final Path FILE_PATH = Paths.get("src/test/resources/hello.txt");
+    private TestRunner testRunner;
+
+    @BeforeEach
+    public void setUp() {
+        testRunner = TestRunners.newTestRunner(EncodeContent.class);
+    }
 
     @Test
     public void testBase64RoundTrip() throws IOException {
```

Comment:
Since you are adding other tests which do not have `public`, please remove `public` from this test and any other test which has `public` since for JUnit 5 `public` is not needed.
```suggestion
    void testBase64RoundTrip() throws IOException {
```

Reply:
Fixed https://github.com/apache/nifi/pull/8417/commits/8c16ea83e7ab83c19f4e4f685b0b30dd0d5edc50